src/api-state/reducer.js

```diff
@@ -0,0 +1,126 @@
+// @flow
+import {
+	API_REQ,
+	API_RESP_SUCCESS,
+	API_RESP_ERROR,
+	API_RESP_FAIL,
+	API_RESP_COMPLETE,
+} from './sync/apiActionCreators';
+import { CACHE_SUCCESS } from './cache/cacheActionCreators';
+
+type ApiState = {
```

Comment:
Yeah it would definitely be worth creating a 'libdef' that is specific to this module - you have to put shared type definitions in a particular place (`/flow-typed/`) and prefix them with `declare` but it's pretty straightforward and would definitely be useful here - will do